docs/source/releases/v1_11_3a0.rst

```diff
@@ -14,6 +14,7 @@ Version 1.11.3a0 (2023-08-19)
 *****************************
 
 * Require sphinx<7.2 due to bugs introduced 2023-08-17 7.2.x release
+* Add .vscode settings repository to "full_install.sh"
```

Comment:
NOTE: Do not merge until add_release_note_to_index has been merged to main, so this PR will be automatically updated to point to main.

Reply:
Done.